### PR TITLE
Make gdal activate script also work in other shells.

### PR DESCRIPTION
**Pull request by cpaulik**
See the related issue and PR at https://github.com/conda/conda-recipes/issues/677 and https://github.com/conda/conda-recipes/pull/679

I've tested this locally in bash and zsh.

### CODE REVIEW

File: recipe/scripts/activate.sh [hunk 1]
```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
 if [[ -z "$GDAL_DATA" ]]; then
-  DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
-  export GDAL_DATA="${DIR}/../../../share/gdal"
+  export GDAL_DATA=$(gdal-config --datadir)
```
Comment (ocefpaf) [inline recipe/scripts/activate.sh]: :+1:

**Comment by ocefpaf** [comment]
Nice addition @cpaulik! Please you bump the build number so this change gets into the release.

**Comment by ocefpaf** [comment]
I cancelled AppvVeyor b/c `gdal` is a heavy build and we don't really need to build it for this change. BTW, once you bump the build number, you can add `[ci skip]` in the commit message so we can merge it ASAP.

**Comment by gillins** [comment]
Should we try to support tcsh also? Just an idle thought...

**Comment by cpaulik** [comment]
I've updated the build number.

**Comment by ocefpaf** [comment]
> Should we try to support tcsh also? Just an idle thought...

Do we have too? With the propper shebang that script will execute correctly in any shell, right?

**Comment by gillins** [comment]
Sourcing doesn't seem to respect the shebang since it must be executed by the same shell. However, I just tested this and `activate` itself doesn't work with tcsh so just ignore my idle thought....

**Comment by ocefpaf** [comment]
Got it. I will investigate if there are any clean solutions to be more inclusive with other shells.

**Comment by ocefpaf** [comment]
@gillins if you agree with the changes here feel free to merge. I will back port it later to `1.11.4`

**Pull request merged**